integration/spark/src/main/scala/org/apache/spark/sql/hive/CarbonAnalysisRules.scala

```diff
@@ -170,6 +171,7 @@ object CarbonIUDAnalysisRule extends Rule[LogicalPlan] {
     logicalplan transform {
       case UpdateTable(t, cols, sel, where) => processUpdateQuery(t, cols, sel, where)
       case DeleteRecords(statement, table) => processDeleteRecordsQuery(statement, table)
+      case ShowPartitions(t) => ShowCarbonPartitionsCommand(t)
```

Comment:
this code is for spark 1.6 version, not spark2, moreover there is no DDLStrategy for spark1.6